installation/linux.md

```diff
@@ -607,13 +597,11 @@ sudo systemctl stop openhab2.service
 # backup current installation with settings
 TIMESTAMP=`date +%Y%m%d_%H%M%S`;
 sudo mv /opt/openhab2 /opt/openhab2-backup-$TIMESTAMP
+```
 
-# download new version (please replace URL)
-cd /tmp
-wget https://openhab.ci.cloudbees.com/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab/target/openhab-2.0.0-SNAPSHOT.zip
-sudo unzip openhab-2.0.0-SNAPSHOT.zip -d /opt/openhab2
-rm openhab-2.0.0-SNAPSHOT.zip
+You may restore these files by moving them back into your openhab folder:
 
+```shell
 # restore configuration and userdata
 sudo cp -arv /opt/openhab2-backup-$TIMESTAMP/conf /opt/openhab2/
 sudo cp -arv /opt/openhab2-backup-$TIMESTAMP/userdata /opt/openhab2/
```

Comment:
Would you like to use a variable for the destination as done here: http://docs.openhab.org/installation/linux.html#backup-and-restore ??